custom_components/powercalc/data/lifx/LIFX Mini Color/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "TP-Link KP125",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 2,
+        "VERSION": "v0.19.12:docker"
+    },
+    "name": "LIFX Mini Color",
+    "standby_power": 0.09,
```

Comment:
Typically zigbee based lights have a standby draw of about 0.2 - 0.3 watt. 
Wifi light are mostly > 0.5 watt. This is because the radio needs more power.
I also find multiple references that the Lifx lights are about 0.5 in standby. 

https://www.reddit.com/r/lifx/comments/eqgfai/power_use/
https://www.futureproofreviews.com/reviews/home-garden/climate-friendly-smart-led-light-bulbs/#:~:text=Drawing%200.5%20W%20in%20standby,light%20bulb%20and%20bridge%20combined.

I think we need to set this to 0.5 watt to get a more realistic figure.
Your powerplug apparently is inaccurate on these really low power levels.

Reply:
Ah, let me change that. Didn't catch this before the action went off.